Gemfile

```diff
@@ -15,10 +15,12 @@ gem 'sass-rails',                 '6.0.0'
 gem 'webpacker',                  '4.2.2'
 gem 'turbolinks',                 '5.2.1'
 gem 'jbuilder',                   '2.10.0'
+# pg был только в продакшне
+gem 'pg',                         '1.2.3'
 gem 'bootsnap',                   '1.4.6', require: false
 
 group :development, :test do
-  gem 'sqlite3', '1.4.2'
+  # удалил гем sqlite3
```

Comment:
Same here. Looks like the project should have Rubocop setup. Let's make it optional task to fix Rubocop offences